amd_openvx_extensions/amd_opencv/CMakeLists.txt

```diff
@@ -117,3 +117,18 @@ else()
 	target_link_libraries(vx_opencv openvx)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fdata-sections")
 endif()
+
+
+if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
+  #include(ROCMHeaderWrapper)
+  # Copy Header files to build directory
+  configure_file("include/vx_ext_opencv.h" "${PROJECT_BINARY_DIR}/include/mivisionx/vx_ext_opencv.h" COPYONLY)
```

Comment:
change "${PROJECT_BINARY_DIR}/include/mivisionx/vx_ext_opencv.h" to
"${PROJECT_BINARY_DIR}/include/vx_ext_opencv.h"